vassal-app/src/main/java/VASSAL/build/GameModule.java

```diff
@@ -1111,6 +1116,31 @@ public void dumpNewListeners() {
     }
   }
 
+  public void backupNewListeners() {
+    if (!isLoadingContinuationSemaphore()) {
+      Collections.copy(keyStrokeSourcesToAdd, keyStrokeSourcesToAddBackup);
+      Collections.copy(keyStrokeListenersToAdd, keyStrokeListenersToAddBackup);
+      Collections.copy(sideChangeListenersToAdd, sideChangeListenersToAddBackup);
+    }
+  }
+
+  public void restoreNewListeners() {
+    if (!isLoadingContinuationSemaphore()) {
+      Collections.copy(keyStrokeSourcesToAdd, keyStrokeSourcesToAddBackup);
+      Collections.copy(keyStrokeListenersToAdd, keyStrokeListenersToAddBackup);
+      Collections.copy(sideChangeListenersToAdd, sideChangeListenersToAddBackup);
+    }
+  }
```

Comment:
Why do backup and restore use the exact same code?

Reply:
that code is no longer in the PR

Reply:
(but good point otherwise haha)